src/website/views/components/forms/checkbox/_examples.pug

```diff
@@ -149,24 +156,30 @@ p.-text
 .example.-mb--3
   .-p--3
     .chi-form__item
-      .chi-checkbox
-        input(type="checkbox" class="chi-checkbox__input" id="checkbox-he1")
-        label(for="checkbox-he1" class="chi-checkbox__label") Checkbox
-        .chi-label__help
-          button.chi-button.-icon.-xs.-flat(id="example__help-button" aria-label="Help" data-target="#example__help-popover")
-            i.chi-icon.icon-circle-info-outline(aria-hidden="true")
-          section.chi-popover.chi-popover--top(id="example__help-popover" aria-modal="true" role="dialog")
-            .chi-popover__content
-              p.chi-popover__text
-                | Helpful information goes here.
-  .example-tabs.-pl--2
+      chi-checkbox#example__help(label='Checkbox')
+        .chi-label__help(slot="chi-checkbox__help")
+          chi-button#example__help_button(type='icon', size='xs', variant='flat', alternative-text='Help')
+            chi-icon(icon='circle-info-outline')
+      chi-popover#example__help_popover(position='top', variant='text', arrow='arrow', reference='#example__help_button') Helpful information goes here.
+  .example-tabs.-pl--
     ul.chi-tabs#example-help
-      li.-disabled
-        a(href=`#webcomponent-help`) Web Component
       li.-active
+        a(href=`#webcomponent-help`) Web Component
+      li
         a(href=`#html-help`) HTML Blueprint
-  .chi-tabs-panel#webcomponent-help
-  .chi-tabs-panel.-active#html-help
+  .chi-tabs-panel.-active#webcomponent-help
+    :code(lang="html")
+      <div class="chi-form__item">
+        <chi-checkbox id="example__help" label="Checkbox">
+          <div class="chi-label__help" slot="chi-checkbox__help">
+            <chi-button id="example__help_button" type="icon" size="xs" variant="flat" alternative-text="Help">
+                <chi-icon icon="circle-info-outline"></chi-icon>
+            </chi-button>
+          </div>
+        </chi-checkbox>
+        <chi-popover id="example__help_popover" position="top" variant="text" arrow="arrow" reference="#example__help_button">Helpful information goes here.</chi-popover>
+      </div>
```

Comment:
@soujanreddy this example is not going to work as it is missing the functional part `<script>...</script>`. You can follow Text input Help example.